src/recordlinker/models/pii.py

```diff
@@ -5,18 +5,27 @@
 import dateutil.parser
 import pydantic
 
-# The Patient features that can be used for comparison.
-FEATURE = typing.Literal[
-    "birthdate",
-    "mrn",
-    "sex",
-    "first_name",
-    "last_name",
-    "line",
-    "city",
-    "state",
-    "zip",
-]
+
+class Feature(enum.Enum):
+    """
+    Enum for the different Patient attributes that can be used for comparison.
+    """
+
+    BIRTHDATE = "birthdate"
```

Comment:
should we change this to `birth_date` as well?

Reply:
I think eventually that change makes sense, because we're already using snake case for `first_name` and `last_name`, however I purposefully didn't want to do that now.

My goal, is to get the new schema and code working in near parallel to the existing schema.  That means, we should be able to use the existing DIBBs basic and enhanced algorithms with both of the schemas.  Right now, that's technically not completely possible, because the matching function signatures have changed between the two schemas, but other than that I'm trying to leave the algorithm definitions intact so we need minimal changes to that config when we move back and forth between testing the old and new schema.  Because the old codebase is still referencing the feature as "birthdate" in that codebase, I choose to leave it as is for now.